packages/components/src/Accordion/AccordionItem.tsx

```diff
@@ -9,6 +9,27 @@ import { SVG, useClassNames, useStateProps } from '@marigold/system';
 import { useAccordionItem } from './useAccordionItem';
 import { Button } from '../Button';
 
+// Select Icon
+// ---------------
+interface ChevronProps {
+  className?: string;
+}
+// icons
+// -----------------
+const ChevronDown = ({ className }: ChevronProps) => (
```

Comment:
Move to own component and reuse in all components? 🤔

Reply:
@sebald 
I did some changes now